tests/FunctionalTestCase.php

```diff
@@ -114,6 +115,74 @@ protected function getServerStorageEngine(ReadPreference $readPreference = null)
         throw new UnexpectedValueException('Could not determine server storage engine');
     }
 
+    protected function isShardedClusterUsingReplicasets()
```

Comment:
It doesn't depend on it, but as PHPLIB doesn't test anything besides standalone, it *is* going to be useful once we move the topologies over from PHPC. Right now, I'm using the Travis topologies from PHPC to test PHPLIB too locally so that we can find these "bugs" in the test skips.